modules/javafx.web/src/main/native/Source/WebCore/Modules/compression/CompressionStreamEncoder.cpp

```diff
@@ -88,6 +88,7 @@ static bool didDeflateFail(int result)
 #if !PLATFORM(JAVA)
     return result != Z_OK && result != Z_STREAM_END && result != Z_BUF_ERROR;
 #endif
+    return true;
```

Comment:
How did this even compile previously? Is this file actually included in our build?